test/cork.c

```diff
@@ -89,10 +89,11 @@ static unsigned
 verify_old_dset_cork(void)
 {
     /* Variable Declarations */
-    hid_t      fid = -1;                             /* File ID */
-    hid_t      did = -1, did2 = -1, did3 = -1;       /* Dataset IDs */
-    hid_t      dcpl = -1, dcpl2 = -1, dcpl3 = -1;    /* Dataset creation property lists */
-    hid_t      sid = -1, sid2 = -1, sid3 = -1;       /* Dataspace IDs */
+    hid_t fid = H5I_INVALID_HID;                                                 /* File ID */
+    hid_t did = H5I_INVALID_HID, did2 = H5I_INVALID_HID, did3 = H5I_INVALID_HID; /* Dataset IDs */
```

Comment:
I still think these should be individual lines, but won't complain if they stay as is.

Reply:
Agreed, but probably the change should start in develop and then be merged down to the other branches?